app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<p>
+  Hello, <%= @org.name %>
+</p>
+
+<p>
+<% if @organization_campaign %>
+  Your current campaign is <%= @organization_campaign.campaign.name %>
+
+  <table>
+    <tr>
+      <th>Assigned Recipients</th>
+      <th>Matched</th>
+      <th>Percent</th>
+    </tr>
+    <tr>
+      <td><%= @organization_campaign.assigned %></td>
+      <td><%= @organization_campaign.matched %></td>
+      <td><strong><%= @organization_campaign.matched_pct %></strong></td>
+    </tr>
+  </table>
+
+<% else %>
+  You are not joined to any campaign at the moment.  Perhaps you should join one!
+<% end %>
+</p>
+
+<%= form_for @organization_campaign_new, 
+    url: {action: :create, controller: :organization_campaigns} do |f| %>
+  Join campaign: 
+  <%= f.hidden_field :organization_id %>
+  <%= f.collection_select(:campaign_id, @joinable_campaigns,:id, :name) %>
+  <%= f.submit "Join" %>
+<% end %>
+
+<% if @joined_campaigns %>
```

Comment:
What's the purpose of this? Is it to view older campaigns? I thought that an organization could only be in one active campaign at a time

If so, I'd probably punt on that as a separate user story and create a separate controller/routes (some type of archive controller)

If not, I'd look at packaging this slightly different as part of a future user story/refactor.

- Make this a `render` instead of an inline form
- Make it slightly smarter, if they have joined more than one campaign show it

Right now it shows a dropdown even though there isn't anything to switch

![image](https://cloud.githubusercontent.com/assets/191564/16543073/dfb21670-4091-11e6-9651-c05485483564.png)